src/org/thoughtcrime/securesms/ProfileSettingsFragment.java

```diff
@@ -233,6 +236,18 @@ public void onSharedChatClicked(int chatId) {
     getActivity().finish();
   }
 
+  private void onVerifiedByClicked() {
+    DcContact dcContact = dcContext.getContact(contactId);
+    if (dcContact.isVerified()) {
+      int verifierId = dcContact.getVerifierId();
+      if (verifierId != 0 && verifierId != contactId) {
```

Comment:
if this gets fixed in core then here we need to change and check if verifierId == SELF_CONTACT_ID

Reply:
let's see what the outcome of https://github.com/deltachat/deltachat-core-rust/issues/4748 will be - if we did sth. wrong or there will be a fix in core. currently we seem to have the same state as desktop, so i think, we can merge this pr in and iterate over in a subsequent pr as needed